test/functional/p2p_compactblocks.py

```diff
@@ -294,7 +294,7 @@ def test_compactblock_construction(self, test_node, use_witness_address=True):
         block.rehash()
 
         # Wait until the block was announced (via compact blocks)
-        wait_until(lambda: "cmpctblock" in test_node.last_message, timeout=30, lock=mininode_lock)
+        self.wait_until(lambda: "cmpctblock" in test_node.last_message, timeout=30, lock=mininode_lock)
```

Comment:
I'm not sure if this is one of the goals of this PR, but in this situation I think it's best to use `test_node.wait_until()`. 

Same with the rest of the test.

Reply:
I was inclined to use the one from `BitcoinTestFramework` since I thought it would be good to keep consistency. However, in this case, since we are waiting for a change in `test_node`, using that one makes more sense, did I get it right? :)

Reply:
Updated.